src/database/queries/sessions.js

```diff
@@ -592,6 +592,15 @@ exports.getUpcomingSessionsFromView = async (
 		const filterClause = filterConditions.length > 0 ? `AND ${filterConditions.join(' AND ')}` : ''
 
 		const saasFilterClause = saasFilter != '' ? saasFilter : ''
+
+		let publicSessionFilter = " OR visibility = '" + common.SESSION_TYPE.PUBLIC + "'"
```

Comment:
@adithyadinesh0412 it should be the type right not visibility , check the session model 
 change From OR to AND condition